pages/alerts.rst

```diff
@@ -312,6 +314,13 @@ Configuration Color
 Custom Message
     The message that will be sent to Slack.  The data described above can be used in this template.
 
+Backlog Message Items (optional)
+    The custom message has a data point named `backlog <http://localhost:8000/pages/alerts.html?highlight=alerts#backlog>`__, which gives
+    the preview of the messages that led to a slack notification.
+    Backlog Message Items will be used to override the Event definition Notification settings field `Message Backlog`
```

Comment:
Again, name should match what is used on the UI (and I think should include the word `Limit` to make the purpose clear).